my_bash_script.sh

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
I think we need a little more logic here. From what I understood is that we will need to create the baseball database, populate it and then run our SQL on it. This looks like it is just a connection?